plugin/src/cli_utils/localpv/lvm/mod.rs

```diff
@@ -1,12 +1,14 @@
-use clap::Parser;
-use plugin::ExecuteOperation;
-pub(crate) mod node;
-pub(crate) mod volume;
 use plugin::resources::utils::OutputFormat;
+use plugin::ExecuteOperation;
 
+use clap::Parser;
 use snafu::Snafu;
 use std::path::PathBuf;
 
+pub(crate) mod node;
+pub(crate) mod supportability;
+pub(crate) mod volume;
```

Comment:
Should these be pub, looking beyond this crate to possible derivations built from this library?

Reply:
No they don't need to be pub, atleast at the moment. There is no other consumer of this at the moment.